### PR TITLE
Minor readme fixes

### PR DESCRIPTION
**Pull request by rarchitgupta**
The README hardly had any changes to be made. The App store description is more or less sufficient and doesn't need to be elaborated upon further. Still a couple of grammatical fixes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,19 +8,19 @@
 
 ### [](https://github.com/pushpinderpalsingh/SpaceDash#a-dashboard-for-space)A Dashboard for Space...🚀🚀🚀 
 
-SpaceDash is an app that **provides information** on all the present, past, and future **Launches and Vehicles of SpaceX**. The SpaceDash app also includes news and updates regarding vehicles and missions.
+SpaceDash is an app that provides information on all the present, past, and future **launches and vehicles of SpaceX**. The SpaceDash app also includes news and updates regarding all such vehicles and missions.
```
Comment (clarkeben) [inline README.md:11]: I am happy with the initial changes to the first sentence, the second does not read well/is not grammatically correct. Please omit **such**.

**Review by clarkeben — CHANGES_REQUESTED** [review]
One minor tweak - everything else has been improved 👍 .

**Comment by rarchitgupta** [comment]
Removed it!

**Comment by pushpinderpalsingh** [comment]
@all-contributors add @rarchitgupta for docs

**Pull request merged**